action/messageHubFeed.js

```diff
@@ -120,6 +156,20 @@ function validateParameters(rawParams) {
         return;
     }
 
+    if(rawParams.kafka_admin_url) {
+        validatedParams.kafka_admin_url = rawParams.kafka_admin_url;
```

Comment:
This is a new parameter that needs to be supplied when creating the trigger. It is part of the Bluemix credentials and will be "magically" present when using `package refresh`.